### ISSUE TITLE
Exception: No image to work with, provide an image or allow the image picker.

### ISSUE THREAD

**Issue by waheedahmadbhatti** [hsbijarniya/image_editor_plus#51]
In Multiple Image editing they give the issue when i select the image.
  Working Correctly for One Image 
## Using Image Picker 
```
   final files = await _fileChooserService.chooseMultipleFiles(
      source: fileSource,
    );
```
## Passing Images to Image Editor
```
   if (fileSource.isImage) {
        List<Uint8List> imageBytesList = [];
        for (var file in files) {
          List<int> bytes = await file.readAsBytes();
          imageBytesList.add(Uint8List.fromList(bytes));
        }

        var editedImages = await Navigator.push(
          context,
          MaterialPageRoute(
            builder: (context) => ImageEditor(
              images: imageBytesList,
              allowMultiple: false,
            ),
          ),
        );

        if (editedImages != null && editedImages is List<Uint8List>) {
          filesCustomData!.addAll(editedImages.map((image) => CustomFileData(file:File.fromRawPath(image) )).toList());
        }
      }
```

**Comment by waheedahmadbhatti** [comment]
## Cose of the Problem 
```
   if (images != null && image == null && !allowCamera && !allowGallery) {
      throw Exception(
          'No image to work with, provide an image or allow the image picker.');
    }
```

Why it's important to Allow the Camera and Gallery 
```
 allowGallery: true,
 allowCamera: true,
```

**Comment by hsbijarniya** [comment]
fixed in v0.2.6
